hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/ContainerData.java

```diff
@@ -321,10 +370,20 @@ public long getWriteBytes() {
 
   /**
    * Increase the number of bytes write into the container.
+   * Also decrement committed bytes against the bytes written.
    * @param bytes the number of bytes write into the container.
    */
   public void incrWriteBytes(long bytes) {
+    long unused = getMaxSize() - getBytesUsed();
```

Comment:
Maybe we need a new method to decrement the bytes  such as named `decrWriteBytes`.

Reply:
there is no requirement for decrWriteBytes currently in any flow.
But I have added a new method decCommittedBytes(delta) in HddsVolume to complement incCommittedBytes(delta). This improves readability and the caller does not have to do a (0 - delta) to decrease commit space.

Reply:
Yes, it's good!